packages/components/src/organisms/Filter/FilterSlider.tsx

```diff
@@ -94,19 +101,23 @@ function FilterSlider({
       </div>
       {footer && (
         <footer data-fs-filter-slider-footer>
-          <Button
-            data-fs-filter-slider-footer-button-clear
-            {...clearBtnProps}
-          />
-          <Button
-            data-fs-filter-slider-footer-button-apply
-            data-testid="filter-slider-button-apply"
-            {...applyBtnProps}
-            onClick={(e) => {
-              applyBtnProps?.onClick?.(e)
-              fadeOut()
-            }}
-          />
+          {clearBtnProps && (
+            <Button
+              data-fs-filter-slider-footer-button-clear
+              {...clearBtnProps}
+            />
+          )}
+          {applyBtnProps && (
+            <Button
+              data-fs-filter-slider-footer-button-apply
+              testId={`${testId}-button-apply`}
```

Comment:
added this testId to differentiate because we are using for both scenario `FilterSlider` and `RegionSlider`, and avoid issues on tests.